main/test/flixcheck/src/SplittableRandom.flix

```diff
@@ -0,0 +1,47 @@
+
+/// Represents a splittable pseudorandom number.
+pub enum SplittableRandom(##java.util.SplittableRandom)
+
+mod SplittableRandom {
```

Comment:
Oh jeg vidste ikke at i har jeres egen wrapper. Men i kan effect annotere ligesom mit library for at undgå IO når i bruger freshWithSeed